URL-API.md

```diff
@@ -4,77 +4,289 @@ MazeMap URL API
 ```
->#####campusid's#####
->1. NTNU
->3. St.Olavs Hospital
->5. UiT
+
+* `campusid` defines the campus in view, in this case `1`.
+
+To specify other parameters you must also specify the campus to provide proper context.
+
+#### Examples of campusids
+
+* 1 = NTNU Gløshaugen
+* 3 = St. Olavs Hospital
+* 4 = OSL
+* 5 = UiT
+
+A full list of campus IDs migth be available in the future.
 
 
-2. Specifying a POI destination
+Specifying a POI (Point of Interest)
 --------------------
-To start mazemap with a specific POI in view, use the similar syntax as when specifying a building. 
+
+#### POIs in general
+
+There are 3 different types of POIs:
+
+1. `sharepoitype`
+2. `starttype`
+3. `desttype`
+
+They can be specified in 3 different ways:
+
+1. `poi` defines a MazeMap POI ID
+2. `point` defines a generic point by longitude, latitude and floor
+3. `identifier` defines a local ID
+
+This defines the format expected by the parameter related to the type of POI.
+
+* `sharepoi` for `sharepoitype`
+* `start` for `starttype`
+* `dest` for `desttype`
+
+The general syntax is as follows:
+
+```
+<sharepoi,start,dest>type=poi&<sharepoi,start,dest>=<MazeMap POI ID>
+<sharepoi,start,dest>type=point&<sharepoi,start,dest>=<longitude>,<latitude>,<floor>
+<sharepoi,start,dest>type=identifier&<sharepoi,start,dest>=<Lydia Building ID>-<Lydia Room ID>
+```
+
+
+#### Specifying a POI destination
+
+To start MazeMap with a specific POI in view:
 
 ```
 http://use.mazemap.com/?campusid=1&desttype=poi&dest=593
 ```
-*   _**desttype**_ defines the type expected for the dest parameter. For a POI, you can must use the word **poi** here.
-*   _**dest**_ defines the specific **POI ID**.
 
+* `desttype` defines the format expected by the `dest` parameter. In this case a POI ID.
+* `dest` defines the specific POI ID.
 
-3. Specifying a generic point destination
---------------------
-To start mazemap with a generic point in view use longitude and lattitude format
+#### Specifying a generic point destination
+
+To start MazeMap with a generic point in view:
 
 ```
 http://use.mazemap.com/?campusid=1&desttype=point&dest=10.4026794,63.4183615,0
 http://use.mazemap.com/?campusid=1&desttype=point&dest=10.4035833,63.4178412,3
 ```
-*   _**desttype**_ defines the type expected for the dest parameter. For a generic point, you can must use the word **point** here.
-*   _**dest**_ defines the geographical point in 4326 comma-separated array as shown below.
-```javascript
-dest=lon,lat,z
-``` 
-The the z value is used to specify the _floor_, if the point lies inside a building. If outside, the _**z**_ parameter can be dropped, or be simply 0.
 
+* `desttype` defines the format expected by the `dest` parameter. In this case a generic point.
+* `dest` defines the geographical point as a comma-separated list `dest=longitude,latitude,z` where `z` is used to specify the _floor_ if the point lies inside a building. If outside, the `z` parameter should be 0, or it can simply be dropped.
```

Comment:
"outside" → "outdoors"

Maybe add a note about `z` being a number that *usually* matches the floor name, but is **not** guaranteed to do so.